### DISCUSSION TITLE
Github student

### DISCUSSION THREAD

**Discussion by GasterWS** [community/community#44465] [GitHub Education]
I've been trying to extend my github student developer pack but whenever i try to open the github sudent site i get aa 502 Bad Gateway error, is this functionality depricated or is it a problem on my side?

**Comment by jge162** [comment]
It's likely that the error you're encountering is not related to your student developer pack status, but rather a problem with the GitHub website. A 502 Bad Gateway error typically indicates that there is an issue with the server that is acting as a proxy for the website. It could be caused by a number of things, including a temporary overload of the server, an issue with the server's configuration, or a problem with the network connection.

If the problem is on your side, you can try clearing your browser cache and cookies, or try accessing the website using a different browser or device. You can also try disabling any browser extensions or VPNs that you may be using, as they can sometimes cause issues with accessing certain websites.

If the problem persists, you can try checking the GitHub status page (https://www.githubstatus.com/) to see if there are any known issues with the website. You can also try reaching out to GitHub support for further assistance.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @GasterWS , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by Jpergon251** [comment]
Hi i have a problem, im trying to get student benefits, i have the proof of my school ID but i cant submit cause its says an error: 'You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.'
Im in my school so i dont know what can i do to fix it?

**Reply by jge162** [reply]
Just post an image of your transcript. Showing your currently enrolled.
